students/Sally_Shen/lesson3/mailroom.py

```diff
@@ -0,0 +1,96 @@
+
+
+'''choms'''
+
+import sys
+
+print("in mailroom")
+'''Writing first name is easier for testing.'''
+donors ={"Fred Gates": [100.90, 200, 50], "Jeff Bazos": [100, 50, 600], "Mark Zuckerberg": [200, 50, 500],
+         "Paul Allen": [300, 40, 200]}
+
+
+def thank_you():
+    response = input("Type your donor name, if don't know, type list to pick")
+    '''If the user types list show them a list of the donor names and re-prompt'''
+    if response == "list":
+        for key in donors.keys():
+            print(key)
+        response = input("Please type your donor name.")
+
+    if response not in donors.keys():
+        donors[response] = []
+
+    newDonation = input("how much to donate?")
+    donors[response].append(float(newDonation))
```

Comment:
so you do turn it into a number here -- good.

but then you use the string version is letter below.

you really want to make it a number first, and use the number both when adding to the donations list and in the letter - that way you can use string formatting in the letter to present it correctly: $200.00, for instance.